include/zephyr/net/net_if.h

```diff
@@ -2113,6 +2166,90 @@ static inline bool net_if_is_up(struct net_if *iface)
  */
 int net_if_down(struct net_if *iface);
 
+/**
+ * @brief Check if interface was brought up by the application or system.
```

Comment:
Perhaps better to say
`Check if interface was brought up by the administrator.`
I think we should not say "system" here.